connect-button/src/main/java/com/ifttt/connect/ui/EventsList.java

```diff
@@ -0,0 +1,12 @@
+package com.ifttt.connect.ui;
+
+import com.squareup.moshi.Json;
+import java.util.List;
+
+class EventsList{
+    @Json(name = "events") private List<AnalyticsEventPayload> events;
```

Comment:
Seems unnecessary as we have the same field name as the JSON key. This should also be final.